cluster/saltbase/salt/flannel/init.sls

```diff
@@ -0,0 +1,42 @@
+flannel-tar:
+  archive:
+    - extracted
+    - user: root
+    - name: /usr/local/src
+    - makedirs: True
+    - source: https://github.com/coreos/flannel/releases/download/v0.5.5/flannel-0.5.5-linux-amd64.tar.gz
```

Comment:
To clarify, I think we can totally put it in a pod, just that there's enough moving in this pr that I'd rather punt to a follow up if i can. A really sick way to solve this would be to not even require the docker restart because our plugins put pods on the new cidr bridge (hand waving).